docs/streams/quickstart.html

```diff
@@ -0,0 +1,249 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<script><!--#include virtual="../js/templateData.js" --></script>
+
+<script id="content-template" type="text/x-handlebars-template">
+  <h1>Kafka Streams Library Quickstart</h1>
```

Comment:
Since this page has already be wrapped under the `Streams` sub-space, there will be a

```
DOCUMENTATION >> STREAMS
```

I think we do not need the "Kafka Streams" prefix or "Kafka Streams Demo" prefix; I'd propose we use the title as `Play with a Streams Application` for this section and for the future tutorial section title it as `Write your own Streams Application`?

If you guys agree I will go ahead to edit-while-merge.